src/v2/Apps/ViewingRoom/Components/ViewingRoomsFeaturedRail.tsx

```diff
@@ -23,18 +24,15 @@ export const ViewingRoomsFeaturedRail: React.FC<ViewingRoomsFeaturedRailProps> =
   }
 
   const carouselItemRender = (
-    {
-      heroImageURL,
-      slug,
-      title,
-      partner,
-      status,
-      distanceToOpen,
-      distanceToClose,
-    },
+    { image, slug, title, partner, status, distanceToOpen, distanceToClose },
     slideIndex: number
   ): React.ReactElement => {
     const tag = getTagProps(status, distanceToOpen, distanceToClose)
+    const heroImageURL = resize(image?.imageURLs?.normalized, {
+      height: 370,
+      width: 280,
+    })
```

Comment:
@oxaudo if 370 x 280 is the final width we expect to display at, I would suggest resizing to 2x that - so 740h x 560w. Doing so ensures that images look nice on high-def displays; here's a cool example: https://www.danrodney.com/blog/2x-1x-demo/ (from https://www.danrodney.com/blog/retina-web-graphics-explained-1x-versus-2x-low-res-versus-hi-res/)

Reply:
oh also should this be `crop`? I think with `resize` we would end up with images that are not exactly `370 x 280` but instead whose longest side is either 370px or 280px.

An example to illustrate:

Here's an image with `resize_to=fill` (which is what resizer's `crop` function does under the hood):
https://d7hftxdivxxvm.cloudfront.net/?src=https://static01.nyt.com/images/2019/05/14/t-magazine/14tmag-sunkim-slide-ME3X/14tmag-sunkim-slide-ME3X-superJumbo.jpg&resize_to=fill&width=400&height=800

It's exactly 400 x 800 px and is cropped as little as possible to make that happen. Whereas, with `resize_to=fit` (which is what resizer's `resize` function uses), it looks like this:
https://d7hftxdivxxvm.cloudfront.net/?src=https://static01.nyt.com/images/2019/05/14/t-magazine/14tmag-sunkim-slide-ME3X/14tmag-sunkim-slide-ME3X-superJumbo.jpg&resize_to=fit&width=400&height=800

As you can see, it's kept the original aspect ratio of the image and resized it to respect the maximum width we passed in.

Reply:
👍